test/item_test.rb

```diff
@@ -3,23 +3,26 @@
 require_relative '../lib/item'
 
 class ItemTest < MiniTest::Test
+  attr_reader :parent, :item
+  def setup
+    sales_engine = SalesEngine.new
+    @parent = ItemRepository.new('test/support/sample_items.csv', sales_engine)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.